File: 3830-find-closest-person/find-closest-person.cs
public class Solution {
    public int FindClosest(int x, int y, int z) {
        int xd = Math.Abs(z - x);
        int yd = Math.Abs(z - y);

        if (xd == yd)
        return 0;
        else if (xd < yd)
        return 1;
        else
        return 2;
    }
}

File: 3830-find-closest-person/find-closest-person.js
var findClosest = function(x, y, z) {
    let xd = Math.abs(z - x);
    let yd = Math.abs(z - y);

    if (xd > yd )
    return 2;
    else if (yd > xd)
    return 1;
    else
    return 0;
    
    
};